### PR TITLE
User docs for editors

### PR DESCRIPTION
**Pull request by EarthlingDavey**
Hey @EmilyHazlehurst , would you mind reviewing when you have time please?

HTML view is at: https://github.com/ministryofjustice/justice-gov-uk/tree/user-docs/user-docs

This is a draft PR because IDK if it's best to have the .md docs in the repo, or have the content in WordPress only.

### CODE REVIEW

**Comment by wilson1000** [comment]
Hi @EarthlingDavey, we can use GitHub Pages to publish the docs. We can add style later if we want to.

We configure the page here: https://github.com/ministryofjustice/justice-gov-uk/settings/pages

**Review by EmilyHazlehurst — COMMENTED** [review]
@EarthlingDavey I was in the middle of this review before you merged the structure changes so I think my comments are being treated as outdated. 

I'll submit this now so they're not lost, but there might be a few more :)

File: user-docs/redirects.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- Be cautions not to create 2 redirects that direct to each other. This will create an loop and cause the page to fail to load.
+- When updating a redirect, be aware that if you change the 'Redirect From' URL, this may result in a 404 error error for the old URL.
+- Similarly, deleting a redirect rule could cause a 404 error for the old URL. It is usually best not to delete a redirect rule, but to update it to point to a new URL instead.
+
+---
+
+## Reporting an issue
+
+To report an issue or to raise a support query, please contact the Central Digital Product Team at: justice-support@digital.justice.gov.uk
```
[the remark this answers is not in the document]
Comment (EarthlingDavey) [reply user-docs/redirects.md]: We could link to the WordPress dashboard... and have the email address there. I'll delete the email now.

**Comment by EmilyHazlehurst** [comment]
@EarthlingDavey, in fact it might be easier for you to publish the docs and I can review them once they're published and finished. That way I'm not adding comments about loads of comments about screenshots, because I saw that in your comment to Deepika you've not added them yet.

From a code perspective it all looks good though :)

**Comment by EarthlingDavey** [comment]
Hey @EmilyHazlehurst , thanks for the review! Sorry for butchering it 😅

I can at least address your points, publish and then ask for another review. Cheers!

**Comment by EmilyHazlehurst** [comment]
@EarthlingDavey @wilson1000 Just a thought, but could these be added to the wiki in a user guide section? https://github.com/ministryofjustice/justice-gov-uk/wiki/Documentation

That way we don't have to push code every time there's an update.

**Comment by EarthlingDavey** [comment]
Hey @EmilyHazlehurst , the wiki does sound like a good option.

For the time being, I'm finding it pretty easy (and enjoyable DX) to edit locally and see the html (hot reloading) in the browser.

There's a couple of other things to consider like: GH pages allowing for branding and a custom domain.

I'll get the content finished then we discuss which way will work best.

**Comment by wilson1000** [comment]
> Just a thought, but could these be added to the wiki in a user guide section?

Hey @EmilyHazlehurst we spoke about this when Davey was at lunch 😄 ... to summarise @EarthlingDavey; GH Pages is the better solution for user docs as this allows us to detach stakeholder support from technical, and we can style a GH Page. We also discussed putting the docs into a dedicated branch called `gh-pages` so we don't keep triggering a deployment build in the main workflow. What do you think?

File: user-docs/block-editor.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+To add a new block, click the `+` icon at the top left of the editor.
+
+Alternatively, press `Enter` at the end of a block to create a new block below it.
+Type forward slash '/' to see a list of block types you can add, start typing to select the one you want.
+
+### Edit a block
+
+To edit a block, click on the block you want to edit. The majority of blocks are intuitive to edit, and you can usually just start typing.
+
```
[the remark this answers is not in the document]
Comment (EarthlingDavey) [reply user-docs/block-editor.md]: I've edited the text, cutting out the intuitive to edit part :)

File: user-docs/block-editor.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+When you're editing a block, you'll see a toolbar appear above the block. This toolbar contains options for formatting the block, and for moving or deleting the block.
+
+Along the right hand side of the editor, you'll see a sidebar with additional block settings. This sidebar will change depending on the block you have selected.
+
+---
```
[the remark this answers is not in the document]
Comment (EarthlingDavey) [reply user-docs/block-editor.md]: That's good to know. Thanks!

**Review by wilson1000 — APPROVED** [review]
Bril 😃

**Comment by EarthlingDavey** [comment]
@EmilyHazlehurst  & @wilson1000 thanks for the review!

I've merged this, even though I still have a bit left to do. It's a good oportunity for me to see ho the gh-pages workflow works (or doesn't 😀).

Re: a gh-pages it sounds like a good shout, but maybe it won't be needed in the future as bulk doc writing like this PR wont happen.

**Pull request merged**